template/src/services/asyncStorage/storage.js

```diff
@@ -0,0 +1,29 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+export const clearAllData = async () => {
+  try {
+    await AsyncStorage.clear()
+  } catch (e) {}
+}
+
+export const setData = async (key, data) => {
+  const stringValue = typeof data === 'string' ? data : JSON.stringify(data)
+  try {
+    await AsyncStorage.setItem(key, stringValue)
+  } catch (e) {}
+}
+
+export const getData = async key => {
```

Comment:
keep the func name is `getString` so we don't need to change too much

Reply:
I think we need use getData name which have more meaningful